lib/Runtime/Base/CrossSiteObject.h

```diff
@@ -223,21 +223,13 @@ namespace Js
     template <typename T>
     void CrossSiteObject<T>::RemoveFromPrototype(ScriptContext * requestContext)
     {
-        if (GetScriptContext() == requestContext)
-        {
-            __super::RemoveFromPrototype(requestContext);
-        }
-        // else do nothing because we never cache cross-context
+        __super::RemoveFromPrototype(GetScriptContext());
```

Comment:
> Can we actually just get rid of these methods, since they only forward to the parent?

There is one change. It forwards but replaced ```requestContext``` to ```GetScriptContext()```.